### ISSUE TITLE
SC 1.4.11 understanding document implies pass for focus indicator when only focus style has not been changed

### ISSUE THREAD

**Issue by mraccess77** [w3c/wcag#2153]
Statement in the document: focus style | Links are required to have a focus indicator by 2.4.7 Focus Visible. Where the focus style of the user-agent is not adjusted on interactive controls (such as links, form fields or buttons), the default focus style is sufficient.

Should be

Statement in the document: focus style | Links are required to have a focus indicator by 2.4.7 Focus Visible. Where any style of the user-agent is not adjusted on interactive controls (such as links, form fields or buttons), the default focus style is sufficient.

**Comment by patrickhlauke** [comment]
@mraccess77 I filed https://github.com/w3c/wcag/pull/2161
